lib/internal/http2/core.js

```diff
@@ -199,6 +199,9 @@ const onServerStreamErrorChannel = dc.channel('http2.server.stream.error');
 const onServerStreamFinishChannel = dc.channel('http2.server.stream.finish');
 const onServerStreamCloseChannel = dc.channel('http2.server.stream.close');
 
+const clientTracingChannels = dc.tracingChannel('http2.client');
```

Comment:
```suggestion
const clientTracingChannels = dc.tracingChannel('http2.client');
```

Should we have a http2.server.session too?

Reply:
We could, but at the moment I'm only tracking streams (http transactions). We could of course add support for sessions (to track actual connections). We can add those later on. At least this way we have the same support as we have for http and fetch.